### ISSUE TITLE
Create example for building a python library with poetry

### ISSUE THREAD

**Issue by helderco** [dagger/dagger#3679]
- [ ] Get library version from VCS

**Comment by helderco** [comment]
This was meant to depend on #3702 so instead of having examples for hatch and poetry the poetry example could be used to port into mage. Decided not to switch to Poetry now so this is out of scope for the milestone, but I'll work on it, just using hatch as part of #3678 and #3682.

**Comment by helderco** [comment]
Ended up switching to Poetry and this is already happening in your own CI though in Go. Maybe I'll add it later but in the examples repo.
